jenkins-scripts/docker/lib/generic-abi-base.bash

```diff
@@ -88,8 +92,14 @@ echo '# BEGIN SECTION: compile and install branch: ${SRC_BRANCH}'
 # Reusing the same building and source directory to save bandwith and
```

Comment:
If we're going to be reusing the same clone, I think we need to remove the new `git remote` in between builds, otherwise it will err if you try to add a new remote on top of an existing one.

Reply:
The initial checkout that persist between builds should be untouched by our scripts. This git operations should affect to the copy generated in https://github.com/ignition-tooling/release-tools/pull/168/files#diff-aad0eef06c1dce2393171b60f2c8563bR68-R70